lazyscribe/experiment.py

```diff
@@ -282,6 +283,12 @@ def log_artifact(
             if artifact.name == name:
                 if overwrite:
                     self.artifacts[index] = artifact_handler
+                    if handler_cls.output_only:
+                        warnings.warn(
+                            f"Artifact '{name}' is added. It is not meant to be read back as Python Object",
+                            UserWarning,
+                            stacklevel=2,
+                        )

```

Comment:
Hm, do you think we should have the warnings on log, save, and load? Maybe we only need it for load

Reply:
I just thought it would be nice to remind users when saving/logging the object in case they actually want to save the original python object, no other valid argument otherwise